<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Synapse Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot; margin-top:0px; margin-bottom:10px; margin-left:0px; margin-right:0px;&quot; cellspacing=&quot;2&quot; cellpadding=&quot;0&quot; bgcolor=&quot;#ffffff&quot;&gt;
&lt;tr&gt;
&lt;td bgcolor=&quot;#ffffff&quot; style=&quot; vertical-align:top; padding-left:5; padding-right:8; padding-top:1; padding-bottom:1;&quot;&gt;
&lt;p style=&quot; margin-top:16px; margin-bottom:1px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica,Arial,sans-serif'; font-size:8pt; font-weight:600; color:#1d1f22;&quot;&gt;Epsilon&lt;/span&gt;&lt;span style=&quot; font-family:'Helvetica,Arial,sans-serif'; font-size:8pt; color:#1d1f22;&quot;&gt; : float&lt;br /&gt;Maximum distance between two samples for them to be considered in the same neighborhood.&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'Helvetica,Arial,sans-serif'; font-size:8pt; font-weight:600; color:#1d1f22;&quot;&gt;Minimum Samples&lt;/span&gt;&lt;span style=&quot; font-family:'Helvetica,Arial,sans-serif'; font-size:8pt; color:#1d1f22;&quot;&gt; : int&lt;br /&gt;Number of samples in a neighborhood for a point to be considered a core point. Includes the point itself.&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'Helvetica,Arial,sans-serif'; font-size:8pt; font-weight:600; color:#1d1f22;&quot;&gt;Minimum Density&lt;/span&gt;&lt;span style=&quot; font-family:'Helvetica,Arial,sans-serif'; font-size:8pt; color:#1d1f22;&quot;&gt; : int&lt;br /&gt;The number of samples (or total weight) in a neighborhood for a point to be considered as a core point. This includes the point itself.&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QComboBox" name="allCombo"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Data Group</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="previewBtn">
         <property name="text">
          <string>Preview Clusters</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Epsilon</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="allChannel"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Channel Column</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="allSimulateDistancesCheck">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Simulate Distances</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QPushButton" name="allClusterBtn">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="allZcCheck">
         <property name="text">
          <string>Include Zc</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="allMinSamples">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Minimum number of points a cluster must contain to be considered&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="allMinDensity">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Minimum points within epsilon that a point must contain to be considered for clustering.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Minimum Samples</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Minimum Density</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="allEpsilon">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum distance between points to consider them in the same cluster&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="value">
          <double>300.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Distance Cutoff</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="cutoffSpin">
         <property name="toolTip">
          <string>Set to -1 for no cutoff</string>
         </property>
         <property name="minimum">
          <double>-1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>5000.000000000000000</double>
         </property>
         <property name="value">
          <double>-1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_12">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Distance Mode</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="allMode">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nearest Neighbor (No Repeats): Iterates over the points in the channel with fewer centroids, collecting the closest pairs of points until all points are used or the next shortest distance exceeds the cutoff distance.&lt;/p&gt;&lt;p&gt;Closest Y to every X: Iterates over every point in the channel with more centroids, appends the shortest distance to a point of a different channel.&lt;/p&gt;&lt;p&gt;All Distances X-&amp;gt;Y: Iterates over the points in the channel with more centroids, appends the distance from every point in channel X to every point in channel Y, if the distance is less than the cutoff.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Nearest Neighbor (No Repeats)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Closest Y to every X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>All Distances X-&gt;Y</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>434</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_Centroids"/>
    <addaction name="actionExport_Distances"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport_Centroids">
   <property name="text">
    <string>Export Centroids</string>
   </property>
  </action>
  <action name="actionExport_Distances">
   <property name="text">
    <string>Export Distances</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
